src/main/java/net/glowstone/entity/projectile/GlowEnderPearl.java

```diff
@@ -0,0 +1,83 @@
+package net.glowstone.entity.projectile;
+
+import static net.glowstone.net.message.play.entity.SpawnObjectMessage.THROWN_ENDERPEARL;
+
+import com.flowpowered.network.Message;
+import java.util.Arrays;
+import java.util.List;
+import net.glowstone.net.message.play.entity.EntityMetadataMessage;
+import net.glowstone.net.message.play.entity.EntityTeleportMessage;
+import net.glowstone.net.message.play.entity.EntityVelocityMessage;
+import net.glowstone.net.message.play.entity.SpawnObjectMessage;
+import org.bukkit.Location;
+import org.bukkit.block.Block;
+import org.bukkit.entity.EnderPearl;
+import org.bukkit.entity.Entity;
+import org.bukkit.entity.LivingEntity;
+import org.bukkit.event.entity.EntityDamageEvent;
+import org.bukkit.event.player.PlayerTeleportEvent;
+import org.bukkit.projectiles.ProjectileSource;
+import org.bukkit.util.Vector;
+
+public class GlowEnderPearl extends GlowProjectile implements EnderPearl {
+    private static final double ENDER_PEARL_DAMAGE = 5.0;
```

Comment:
Add a line between the class declaration and this field.

Reply:
Done.